components/server-side-render/index.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * External dependencies.
+ */
+import { isEqual } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	Component,
+	RawHTML,
+} from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+export class ServerSideRender extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			response: null,
+		};
+	}
+
+	componentDidMount() {
+		this.fetch( this.props );
+	}
+
+	componentWillReceiveProps( nextProps ) {
+		if ( ! isEqual( nextProps, this.props ) ) {
+			this.fetch( nextProps );
+		}
+	}
+
+	fetch( props ) {
+		this.setState( { response: null } );
+		const { block } = props;
+		const attributes = Object.assign( {}, props );
+
+		// Delete 'block' from attributes, only registered block attributes are allowed.
+		delete attributes.block;
+
+		let apiURL = this.getQueryUrlFromObject( {
+			attributes: attributes,
+		} );
+
+		apiURL = '/gutenberg/v1/block-renderer/' + block + '?' + apiURL;
+
+		return wp.apiRequest( { path: apiURL } ).then( response => {
+			if ( response && response.rendered ) {
+				this.setState( { response: response.rendered } );
+			}
+		} );
+	}
+
+	getQueryUrlFromObject( obj, prefix ) {
+		const str = [];
+		let param;
+		for ( param in obj ) {
+			if ( obj.hasOwnProperty( param ) ) {
```

Comment:
I think it would probably be better to do:

```js
for ( const [ param, value ] of  Object.entries( obj )  ){
```

Then there is no need for `hasOwnProperty` and no no need for `value = obj[ param ]` below.